src/components/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { StyledRouterLink } from "components/styles";
+import { H3, P1 } from "components/Typography";
+import { size } from "constants/tokens";
+
+const { gray } = uiColors;
+export interface Breadcrumb {
+  text: string;
+  to?: string;
+  onClick?: () => void;
+}
+interface BreadcrumbsProps {
+  breadcrumbs: Breadcrumb[];
+}
+const Breadcrumbs: React.VFC<BreadcrumbsProps> = ({ breadcrumbs }) => (
+  <Container>
+    {breadcrumbs.map((breadCrumb, index) => (
+      <Fragment
+        breadCrumb={breadCrumb}
+        key={`breadCrumb-${breadCrumb.text}`}
+        active={breadcrumbs.length - 1 === index}
+      />
+    ))}
+  </Container>
+);
+
+interface FragmentProps {
+  breadCrumb: Breadcrumb;
```

Comment:
[nit] since we're not camel casing anywhere else
```suggestion
  breadcrumb: Breadcrumb;
```